src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -575,7 +474,7 @@ else if (getBuildingLevel() <= WOOD_HUT_LEVEL)
     /**
      * AbstractBuildingWorker View for clients.
      */
-    public static class View extends AbstractBuildingHut.View
+    public static class View extends AbstractBuildingView
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 83 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)